tox.ini

```diff
@@ -23,29 +19,37 @@ covercmd =
 
 [grpc]
 deps =
-    grpcio >= 0.14.0
-    google-gax >= 0.12.1
-    gax-google-pubsub-v1 >= 0.7.10
-    gax-google-logging-v2 >= 0.7.10
+    grpcio >= 1.0rc1
+    google-gax >= 0.12.3, < 0.13dev
+    gax-google-pubsub-v1 >= 0.7.12, < 0.8dev
+    grpc-google-pubsub-v1 >= 0.7.12, < 0.8dev
+    gax-google-logging-v2 >= 0.7.12, < 0.8dev
+    grpc-google-logging-v2 >= 0.7.12, < 0.8dev
 
-[testenv:py27]
-basepython =
-    python2.7
+[docs]
+deps =
+    {[testenv]deps}
+    Sphinx
+    sphinx_rtd_theme
+
+[testenv]
 commands =
     nosetests
 deps =
-    {[testenv]deps}
+    {[testing]deps}
     {[grpc]deps}
+
+[testenv:py26]
```

Comment:
Ok, I was confused on how envlist actually worked with the tox default environments. Thanks @tseaver!